modelling/src/main/java/org/axonframework/modelling/saga/AbstractSagaManager.java

```diff
@@ -161,25 +164,27 @@ protected boolean matchesSegment(Segment segment, String sagaId) {
      * Returns the Saga Initialization Policy for a Saga of the given {@code sagaType} and {@code event}. This policy
      * provides the conditions to create new Saga instance, as well as the initial association of that saga.
      *
-     * @param event The Event that is being dispatched to Saga instances
+     * @param event   The Event that is being dispatched to Saga instances
+     * @param context
```

Comment:
As we will surely move these classes to the legacy/migration package, this new parameter needs some JavaDoc.